### DISCUSSION TITLE
blog/lazyvim-project-specific-settings

### DISCUSSION THREAD

**Discussion by giscus** [kezhenxu94/kezhenxu94.github.io#46] [General]
# blog/lazyvim-project-specific-settings

Developers usually organize their configuration files in a dotfiles repository, while in some cases when you work on specific projects, you have special settings for these projects but you don't want to mess up your dotfiles, in this post I'll show you how to set per-project local configurations without changing your dotfiles.

https://kezhenxu94.me/blog/lazyvim-project-specific-settings

**Comment by ChrisGVE** [comment]
Great post! I have a question, do you know if it is possible to permanently trust the `.lazy.lua` file, to avoid having to trust it each time I open a file in the project?

**Reply by kezhenxu94** [reply]
As far as I know, you don’t need to trust it each time you open the project, only when the .lazy.lua is changed, once you trust it you only need to trust again next time you modify .lazy.lua.

**Comment by bvdeenen** [comment]
I'd suggest setting project specific tabstops, line-endings and such in an [editorconfig file](https://neovim.io/doc/user/editorconfig.html).

This is enabled by default, and will make sure that if/when you share a project with others that are not using neovim (maybe vim, vscode, IntelliJ, ...) they will likely use the same basic settings.

**Reply by kezhenxu94** [reply]
Yes I know editconfig file is a good thing, but the methods introduced in this post is for working on projects that not owned by me, and not to mess up existing code styles. All projects owned by me are configured a formatter/linter to do formatting at build time or compile time
